governance.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Governance 1.0 DRAFT WIP
+description: Governance 1.0 discussion draft
+---
+# Governance 1.0 `DRAFT WIP`
+
+This document contains a work in progress (WIP) draft of the initial governance of the Op1st project (Governance 1.0).
+
+It also has a roadmap for additions coming in the next iteration of governance (Governane 1.1).
+
```

Comment:
What is the process for SIG chair/vice chair elections?

Reply:
yes, should be defined in the SIG creation process- but not here in the overview section.
and maybe we can postpone this detail for v1.1

Reply:
> yes, should be defined in the SIG creation process- but not here in the overview section. and maybe we can postpone this detail for v1.1

We must postpone this kind of detail for a few reasons:

* The process for choosing leaders is an important decision the community needs to make collectively, but we can't do that until we have ratified and are using a process for making decisions as a community. Governance 1.0 is focused on enabling us to make decisions, and tries to make as few decisions on behalf of the community in the 1.0.
* We want to preserve the capability for SIGs to self-organize in different ways, at least until we have a way for the community to decide why and how to restrict that.

We can definitely put in some kind of example/try this language so the initial SIGs can kickstart and refine.